coretex/cli/commands/task.py

```diff
@@ -0,0 +1,88 @@
+#     Copyright (C) 2023  Coretex LLC
+
+#     This file is part of Coretex.ai
+
+#     This program is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU Affero General Public License as
+#     published by the Free Software Foundation, either version 3 of the
+#     License, or (at your option) any later version.
+
+#     This program is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU Affero General Public License for more details.
+
+#     You should have received a copy of the GNU Affero General Public License
+#     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Optional
+from pathlib import Path
+
+import sys
+import logging
+
+import click
+
+from ..modules.user import initializeUserSession
+from ..modules.utils import onBeforeCommandExecute
+from ..modules.project_utils import isProjectSelected
+from ..._task import TaskRunWorker, LoggerUploadWorker, executeProcess
+from ..._task.local.task_config import readTaskConfig
```

Comment:
This should be imported at the same line as `TaskRunWorker`, `LoggerUploadWorker`, and `executeProcess`